docs/resources/aws_ecs_service.md

```diff
@@ -0,0 +1,73 @@
+---
+title: About the aws_ecs_service Resource
+platform: aws
+---
+
+# aws\_ecs\_service
+
+Use the `aws_ecs_service` InSpec audit resource to test properties of a single specific Athena Work Group.
+
+The AWS::ECS::Service resource creates an Amazon Elastic Container Service (Amazon ECS) service that runs and maintains the requested number of tasks and associated load balancers.
+
+## Syntax
+
+Ensure that a services exists.
+    describe aws_ecs_service(cluster: "default", services: ["arn:aws:ecs:us-east-1:012345678910:service/ecs-simple-service"]) do
+      it { should exist }
+    end
+
+## Parameters
+
+`cluster, services` _(required)_
+
+For additional information, see the [AWS documentation on ECS Service](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-ecs-service.html).
+
+## Properties
+
+| Property | Description|
+| --- | --- |
+| services | A list of services to describe. You may specify up to 10 services to describe in a single operation. |
```

Comment:
Is this a list of service ARNs or can it be the service name?

Reply:
Hi Ian, It can either be Service ARN or Service name
Updating 
Thanks